src/plugins/login/loginPlugin.ts

```diff
@@ -11,7 +11,7 @@ export class AzureLoginPlugin {
     this.provider = (this.serverless.getProvider('azure') as any) as AzureProvider;
 
     this.hooks = {
-      'before:deploy:initialize': this.login.bind(this)
+      'package:initialize': this.login.bind(this)
```

Comment:
When using `before:deploy:initialize`, SLS spits out a deprecation warning and instructs to use `package:initialize`